scripts/ci/generate_ci_script.py

```diff
@@ -41,14 +41,38 @@
 from ros_buildfarm.templates import expand_template
 
 
+class _BuildToolArgsHelper(argparse.Action):
+
+    def __call__(self, parser, namespace, values, option_string=None):
+        setattr(namespace, self.dest, values)
+        if (
+            getattr(namespace, 'rosdistro_name', None) is not None and
+            getattr(namespace, 'config_url', None) is not None and
+            getattr(namespace, 'ci_build_name', None) is not None
+        ):
+            try:
+                config = get_config_index(namespace.config_url)
+                build_files = get_ci_build_files(config, namespace.rosdistro_name)
+                build_file = build_files[namespace.ci_build_name]
+
+                parser.epilog = '\n'.join([
+                    (parser.epilog or '') + 'default build tool arguments:',
```

Comment:
If the epilog is a non-empty string which doesn't end with a newline there doesn't seem to be a `\n` before the custom help text follows.

Reply:
0d457c2